### ISSUE TITLE
No NodeSelector causes NPE

### ISSUE THREAD

**Issue by tlawrie** [boomerang-io/roadmap#290]
### Describe the bug

Running Flow with no node selector causes a NPE

### What project does this issue relate to?

Boomerang Flow

### To Reproduce

1. Install Flow with no node selector
2. Execute a workflow with at least 1 task
3. See exception

### Environment

Flow 3.4.1

### Additional Contenxt

```
[2021-11-14 01:37:04.759] [ERROR] [32] ap.ca.co.Co.[T.[l.[/.[dispatcherServlet] : Servlet.service() for servlet [dispatcherServlet] in context with path [] threw exception [Request processing failed; nested exception is java.lang.NullPointerException] with root cause

java.lang.NullPointerException: null
	at io.boomerang.kube.service.TektonServiceImpl.createTaskRun(TektonServiceImpl.java:300) ~[classes!/:2.4.3]
	at io.boomerang.service.TaskServiceImpl.executeTaskTemplate(TaskServiceImpl.java:95) ~[classes!/:2.4.3]
	at io.boomerang.service.TaskServiceImpl.executeTask(TaskServiceImpl.java:74) ~[classes!/:2.4.3]
	at io.boomerang.controller.TaskController.executeTask(TaskController.java:21) ~[classes!/:2.4.3]
```

**Comment by tlawrie** [comment]
Fixed in 6.4.5

**Comment by morarucostel** [comment]
Verified it today in one of our clusters, looks OK.
